tools/cxbe/Main.cpp

```diff
@@ -54,6 +62,40 @@ int main(int argc, char *argv[])
         szXbeTitle[40] = '\0';
     }
 
+    // interpret title id
+    if(szXbeTitleID[0])
+    {
+        bool hex = true;
+        for(int i = 0; szXbeTitleID[i]; ++i)
+        {
+            if(szXbeTitleID[i] == '-')
+            {
+                hex = false;
+                break;
+            }
+        }
+        if(hex)
+        {
+            sscanf(szXbeTitleID, "%x", &dwTitleId);
+        }
+        else
+        {
+            char titlechar[2];
+            unsigned titleno;
+            if (sscanf(szXbeTitleID, "%c%c-%u", &titlechar[0], &titlechar[1], &titleno) != 3)
```

Comment:
This line doesn't follow clang-format rules.

Reply:
I'll see about doing a rebase and running clang-format on all the commits